packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -86,23 +101,24 @@ export class BaseButton extends BaseComponent<IButtonProps, {}> implements IButt
     }
   }
 
-  protected onRenderContent(tag, buttonProps): JSX.Element {
+  protected onRenderContent(tag: any, buttonProps: IButtonProps): JSX.Element {
     return React.createElement(
       tag,
       buttonProps,
-      this.onRenderIcon(),
-      this.onRenderLabel(),
-      this.onRenderDescription(),
-      this.onRenderAriaDescription(),
-      this.onRenderChildren()
-    );
+      React.createElement('div', { className: css(styles.flexContainer, this.classNames.flexContainer) },
+        this.onRenderIcon(),
+        this.onRenderLabel(),
+        this.onRenderDescription(),
+        this.onRenderAriaDescription(),
+        this.onRenderChildren()
+      ));
   }
 
   protected onRenderIcon() {
     let { icon } = this.props;
 
     return icon ? (
-      <span className={ `${this._baseClassName}-icon` }>
+      <span className={ css(`${this.classNames.base}-icon`, this.classNames.icon) }>
```

Comment:
This seems really redundant, this.Classname.base-icon and this.classNames.icon. 'Icon' is the key here, is there a way to simplify it?